tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "baseUrl": ".",
+    "baseUrl": "./",
```

Comment:
I put this back to what it was originally just to see if that was part of the problem, and I figured why not leave it